bullyBot.py

```diff
@@ -0,0 +1,47 @@
+#bullyBot.py
+import os
+import discord
+from discord.ext import commands
+from dotenv import load_dotenv
+
+load_dotenv()
+TOKEN = os.getenv('DISCORD_TOKEN')
+GUILD = os.getenv('DISCORD_GUILD')
+
+bot = commands.Bot(command_prefix='!')
+
+@bot.event
+async def on_ready():
+    for guild in bot.guilds:
+        if guild.name == GUILD:
+            break
+
+    print(
+        f'{bot.user.name} is connected to the following guild:\n'
+        f'{guild.name}(id: {guild.id})'
+    )
+
+@bot.command(name='bully', help='Creates a new channel for bullying a new member')
+async def on_message(ctx, victim):
+    
+
+
+    victim=victim.lower().strip()
+    channel_name=f'making-fun-of-{victim}'
+    guild = ctx.guild
+    existing_channel = discord.utils.get(guild.channels, name=channel_name)
+    bullying_category = discord.utils.get(guild.categories, name='bullying')
+
+    if not existing_channel:
+        response = f'Good choice, {victim} is due for a bullying'
+        #print(f'Creating a new channel for the purpose of bullying {victim}.')
+        await guild.create_text_channel(channel_name, category=bullying_category)
+
+    else:
+        response = f'Hey now, we already have a channel for bullying {victim}'
```

Comment:
TODO: lets pull each response from an an array of possible responses, giving the bot more than one way to respond to respond to each event, which gives it a bit more personality.

Reply:
I like this idea.

Reply:
Added several responses

Reply:
niiiice +1